config/application.yml.default

```diff
@@ -124,8 +124,8 @@ idv_available: true
 idv_contact_phone_number: (844) 555-5555
 idv_max_attempts: 5
 idv_min_age_years: 13
-idv_acuant_sdk_version_default: '11.8.0'
```

Comment:
Does this most people have been running an outdated version of Acuant in their local environments? Should it have been 11.8.1 when we added that version?

Reply:
No because we were explicitly setting these config values in the a/b test in environments where we did the test. We wanted local dev people to use the latest stable one